security.sh

```diff
@@ -1,14 +1,11 @@
 #!/usr/bin/env bash
 echo ${TEST_URL}
+zap-api-scan.py -t ${TEST_URL}/v2/api-docs -f openapi -S -d -u ${SecurityRules} -P 1001 -l FAIL
+echo "ZAP has successfully started"
+curl --fail http://0.0.0.0:1001/OTHER/core/other/jsonreport/?formMethod=GET --output report.json
 export LC_ALL=C.UTF-8
 export LANG=C.UTF-8
-zap-api-scan.py -t ${TEST_URL}/v2/api-docs -f openapi -s -S -d -u ${SecurityRules} -P 1001 -l FAIL
-
-cat zap.out
-echo "ZAP has successfully started"
 zap-cli --zap-url http://0.0.0.0 -p 1001 report -o /zap/api-report.html -f html
-mkdir -p functional-output
-chmod a+wx functional-output
-curl --fail http://0.0.0.0:1001/OTHER/core/other/jsonreport/?formMethod=GET --output report.json
+zap-cli --zap-url http://0.0.0.0 -p 1001 alerts -l Medium --exit-code False
```

Comment:
Any reason why we are looking for Medium.  I think it should be alright to do it until High level .  Also are you sure you removing functional-output directory creation it will work?

Reply:
Reviewer comments has been addressed